lib/cchain/provider/provider_test.go

```diff
@@ -230,7 +234,7 @@ func (f *testFetcher) Fetch(ctx context.Context, chainVer xchain.ChainVersion, f
 
 	f.fetched += len(resp)
 
-	return resp, nil
+	return resp, cachedHeight, nil
```

Comment:
I don't understand why we do this, why not just ignore it and return 0 everywhere?